lib/OIDC/RP/AuthenticationRequest.php

```diff
@@ -90,60 +90,77 @@ public function getRedirectURL(string $authorization_endpoint, array $acr, array
 
         $userinfo_claims = array();
         foreach ($user_attributes as $a) {
-            $userinfo_claims["https://attributes.spid.gov.it/" . $a] = null;
+            $userinfo_claims[$a] = array("essential" => true);
         }
 
         $claims = array(
             "id_token" => array(
-                "nbf" =>  array( "essential" => true ),
-                "jti" =>  array( "essential" => true )
+                "family_name" =>  array( "essential" => true ),
+                "given_name" =>  array( "essential" => true )
             ),
             "userinfo" => $userinfo_claims
         );
 
+        // $request = array(
+        //     "jti" => Util::uuidv4(),
+        //     "iss" => $client_id,
+        //     "sub" => $client_id,
+        //     "aud" => explode(" ", $this->config['aud']),
+        //     "iat" => strtotime("now"),
+        //     "exp" => strtotime("+180 seconds"),
+        //     "client_id" => $client_id,
+        //     "response_type" => $response_type,
+        //     "scope" => $scope,
+        //     "code_challenge" => $code_challenge,
+        //     "code_challenge_method" => $code_challenge_method,
+        //     "nonce" => $nonce,
+        //     "prompt" => $prompt,
+        //     "redirect_uri" => $redirect_uri,
+        //     "acr_values" => implode(" ", $acr_values),
+        //     "claims" => $claims,
+        //     "state" => $state
+        // );
+        $iat = strtotime("now");
+        $exp = strtotime("+1 hour");
         $request = array(
-            "jti" => 'spid-cie-php-oidc_' . uniqid(),
             "iss" => $client_id,
-            "sub" => $client_id,
-            "aud" => array($client_id),
-            "iat" => strtotime("now"),
-            "exp" => strtotime("+180 seconds"),
-            "client_id" => $client_id,
+            "scope" => $scope, #explode(" ", $scope),
+            "redirect_uri" => $redirect_uri,
             "response_type" => $response_type,
-            "scope" => explode(" ", $scope),
-            "code_challenge" => $code_challenge,
-            "code_challenge_method" => $code_challenge_method,
             "nonce" => $nonce,
-            "prompt" => $prompt,
-            "redirect_uri" => $redirect_uri,
-            "acr_values" => $acr_values,
+            "state" => $state,
+            "client_id" => $client_id,
+            "acr_values" => implode(" ", $acr_values),
+            "iat" => $iat,
+            "exp" => $exp,
+            "jti" => Util::uuidv4(),
```

Comment:
verificare se 
'spid-cie-php-oidc_' . Util::uuidv4()
non comporta problemi

Reply:
nessun problema, in realtà mi conformavo a spid-cie-oidc-django e spid-cie-oidc-java come formato default per jti

Reply:
ok, allora lasciamo:
'spid-cie-php-oidc_' . Util::uuidv4()